plugins/flatpak/gs-flatpak.c

```diff
@@ -1524,6 +1533,48 @@ gs_flatpak_add_updates (GsFlatpak *self, GsAppList *list,
 	return TRUE;
 }
 
+static gboolean
+is_endless_app (GsApp *app)
+{
+	return g_str_has_prefix (gs_app_get_id (app), "com.endlessm.");
```

Comment:
Nitpick: Might want to add `gs_app_get_kind (app) == AS_APP_KIND_APP`. `get_app_update_priority()` currently calls this method in a way that this doesn’t matter, but it might matter in future.

Reply:
That check would have to be done outside, as this function is also used inside the `is_endless_content_runtime`. `app` here means GsApp, not Flatpak app.